src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
+                "age=" + age
```

Comment:
```suggestion
        return "Cat{ age=" + age
```
same for other lines